addOns/automation/src/main/resources/org/zaproxy/addon/automation/resources/Messages.properties

```diff
@@ -10,6 +10,179 @@ automation.cmdline.autogenmax.help = Generate template automation file with all
+automation.dialog.addrule.rule = Rule:
+automation.dialog.addrule.threshold = Threshold:
+automation.dialog.addrule.strength = Strength:
+automation.dialog.addrule.error.defaults = Either the Strength or the Threshold must be non Default.\nTo set both to Default delete the rule.
+
+automation.dialog.alerttest.title = Alert Test
+automation.dialog.alerttest.action = Action:
+automation.dialog.alerttest.ruleid = Scan Rule Id:
+automation.dialog.alerttest.alertname = Alert Name (regex):
+automation.dialog.alerttest.url = URL (regex):
+automation.dialog.alerttest.method = Method (regex):
+automation.dialog.alerttest.attack = Attack (regex):
+automation.dialog.alerttest.param = Param (regex):
+automation.dialog.alerttest.evidence = Evidence:
+automation.dialog.alerttest.confidence = Confidence:
+automation.dialog.alerttest.risk = Risk:
+automation.dialog.alerttest.other = Other Info (regex):
+automation.dialog.alerttest.onfail = On Fail:
+
+automation.dialog.all.name = Job Name:
+
+automation.dialog.ascan.summary = Context: {0}
+automation.dialog.ascan.title = Active Scan Job
+automation.dialog.ascan.context = Context:
+automation.dialog.ascan.defaultthreshold = Default Threshold:
+automation.dialog.ascan.defaultstrength = Default Strength:
+automation.dialog.ascan.policy = Policy:
+automation.dialog.ascan.maxruleduration = Max Rule Duration (in mins):
+automation.dialog.ascan.maxscanduration = Max Scan Duration (in mins):
+automation.dialog.ascan.advanced = Show Advanced Options:
+automation.dialog.ascan.remove.confirm = Are you sure you want to remove this rule?
+
+automation.dialog.ascan.delayinms = Delay In MS:
+automation.dialog.ascan.threads = Threads Per Host:
+automation.dialog.ascan.addquery = Add Query Parameter:
+automation.dialog.ascan.handleanticsrf = Handle Anti CSRF Tokens:
+automation.dialog.ascan.injectid = Inject Plugin Id:
+automation.dialog.ascan.scanheaders = Scan 
+
+automation.dialog.ascan.tab.adv = Advanced
+automation.dialog.ascan.tab.policydefaults = Policy Defaults
+automation.dialog.ascan.tab.policyrules = Policy Rules
+
+automation.dialog.ascan.table.header.id = Id
+automation.dialog.ascan.table.header.name = Name
+automation.dialog.ascan.table.header.threshold = Threshold
+automation.dialog.ascan.table.header.strength = Strength
+
+automation.dialog.button.add = Add
+automation.dialog.button.modify = Modify
+automation.dialog.button.remove = Remove
+
+automation.dialog.contexts.table.header.name = Name
+
+automation.dialog.context.tab.context = Context
+automation.dialog.context.tab.include = Include
+automation.dialog.context.tab.exclude = Exclude
+	
+automation.dialog.context.title = Context
+automation.dialog.context.name = Name:
+automation.dialog.context.urls = URLs:
+automation.dialog.context.include = Include RegExs:
+automation.dialog.context.exclude = Exclude RegExs:
+automation.dialog.context.error.badname = You must supply a name
+automation.dialog.context.error.badurl = Invalid URL: ${0}
```

Comment:
Is the `$` on purpose? (Same for following strings.)

Reply:
no :P